src/api.js

```diff
@@ -10,9 +11,18 @@ class Api {
             method,
             body,
             credentials: 'include',
-        }).then((res) => {
+            redirect: 'manual',
```

Comment:
Not sure we need this: If we're explicitly using `res.json()`, shouldn't we also explicity set the `Content-Type` header to `application/json`?

Reply:
I meant the sentence following the colon. But if this defaults to `application/json`, then we're fine (although I don't see where the default is applied?)